app/src/main/java/org/mozilla/tv/firefox/ScreenControllerStateMachine.kt

```diff
@@ -18,7 +20,7 @@ class ScreenControllerStateMachine(private val currentUrlIsHome: () -> Boolean)
         ADD_OVERLAY, REMOVE_OVERLAY, ADD_POCKET, REMOVE_POCKET, ADD_SETTINGS, REMOVE_SETTINGS, EXIT_APP, NO_OP
     }
 
-    var currentActiveScreen: ActiveScreen = ActiveScreen.NAVIGATION_OVERLAY
+    var currentActiveScreen: ActiveScreen = ActiveScreen.POCKET
```

Comment:
Why does this default to `Pocket`?  Shouldn't we always init on `Overlay`?

Reply:
oops - well it gets removed in the state machine commit 😅